DEVELOPER.md

```diff
@@ -250,7 +250,7 @@ location__wireless_profiles__to_merge:
       - mode: ap
         ssid: Private Wifi
         encryption: psk2
-        key: 'file:/root/wifi_pass'
+        key: 'file:/root/wifi_pass' # the location of the file containing the wifi password
```

Comment:
please use 2 spaces in front of the comment, otherwise linting will fail if you copy this code